how-to/integrate-with-snap/README.md

```diff
@@ -0,0 +1,53 @@
+![OpenFin Workspace Example Application -- Adding your application to Storefront](../../assets/OpenFin-Workspace-Starter.png)
```

Comment:
Small thing. The readme says adding your application to storefront.

Reply:
Fixed and the one in the other readme